src/terminal/adapter/adaptDispatch.cpp

```diff
@@ -1473,23 +1473,53 @@ bool AdaptDispatch::SetLineRendition(const LineRendition rendition)
 // - True if handled successfully. False otherwise.
 bool AdaptDispatch::DeviceStatusReport(const DispatchTypes::StatusType statusType, const VTParameter id)
 {
+    constexpr auto GoodCondition = L"0";
+    constexpr auto PrinterNotConnected = L"?13";
+    constexpr auto UserDefinedKeysNotSupported = L"?23";
+    constexpr auto UnknownPcKeyboard = L"?27;0;0;5";
+    constexpr auto LocatorNotConnected = L"?53";
+    constexpr auto UnknownLocatorDevice = L"?57;0";
+    constexpr auto TerminalReady = L"?70";
+    constexpr auto MultipleSessionsNotSupported = L"?83";
```

Comment:
If you want to save a couple bytes in the binary we could either modify `_DeviceStatusReport` to accept a raw pointer or we could make all of these `static constexpr std::wstring_view` and pass them by const-reference. (...since the x64 ABI can't pass structs via registers which would make the switch-case quite fat due to all the SSE2 stack copies.)